examples/jsm/exporters/USDZExporter.js

```diff
@@ -198,6 +198,10 @@ function buildHeader() {
 
 function buildSceneStart( options ) {
 
+	const alignment = options.ar !== undefined ? `
+		token preliminary:anchoring:type = "${options.ar.anchoring.type}"
+		token preliminary:planeAnchoring:alignment = "${options.ar.planeAnchoring.alignment}"
+	` : '';
```

Comment:
There is already the `quickLookCompatible` property. Could it also be an option to do this:
```js
const alignment = options.quickLookCompatible === false ? `
		token preliminary:anchoring:type = "${options.ar.anchoring.type}"
		token preliminary:planeAnchoring:alignment = "${options.ar.planeAnchoring.alignment}"
	` : '';
```
If that does not work, maybe a new boolean is easier to understand than `ar: undefined`. Something like `quickLookAutoAnchoring: true`?

Reply:
I don't think we can use the existing quickLookCompatible here. You may want quickLookCompatible=true for the material fix but still want the anchoring to be floor (alignment horizontal) only.
Another flag quickLookAutoAnchoring may be better indeed, but that one is not a good name imo, it's not really specific to QuickLook here, as far as I know those properties are read for Reality Composer, and probably other softwares that can load usdz. "Auto" in the name is confusing, not sure what that does. I propose `includeAnchoringProperties` as a better name. What do you think?

Reply:
`includeAnchoringProperties` sounds good.

Reply:
Ok, I'll make the changes tomorrow. With `includeAnchoringProperties: true` as the default in options.

Reply:
Ok that's done. I also retested my model-viewer build with it.